eng/install-native-dependencies.sh

```diff
@@ -47,16 +47,8 @@ elif [ "$1" = "OSX" ] || [ "$1" = "tvOS" ] || [ "$1" = "iOS" ]; then
         exit 1;
     fi
 elif [ "$1" = "Android" ]; then
-    if [ -z "${ANDROID_OPENSSL_AAR}" ]; then
-        echo "The ANDROID_OPENSSL_AAR variable is not set. OpenSSL will not be installed."
-        exit 0;
-    fi
-    if [ -d "${ANDROID_OPENSSL_AAR}" ]; then
-        exit 0;
-    fi
-    OPENSSL_VER=1.1.1g-alpha-1
-    curl https://maven.google.com/com/android/ndk/thirdparty/openssl/${OPENSSL_VER}/openssl-${OPENSSL_VER}.aar -L --output /tmp/openssl.zip
-    unzip /tmp/openssl.zip -d "${ANDROID_OPENSSL_AAR}" && rm -rf /tmp/openssl.zip
+    # nothing to do here
+    exit 0
```

Comment:
Can we just stop handling `Android` as an argument and make global-build-job.yml stop calling this for Android?
https://github.com/dotnet/runtime/blob/61889c8e8926aed2a22000c312baa0a7969cc927/eng/pipelines/common/global-build-job.yml#L111-L112

Reply:
Didn't see that.  I like your idea better!